.storybook/preview.jsx

```diff
@@ -22,13 +22,7 @@ export const parameters = {
     'storybook/docs/panel': { index: -1 }
   },
   docs: {
-    page: () => (
-      <>
-        <Title />
-        <Description />
-        <Stories includePrimary />
-      </>
-    )
+    toc: true,
```

Comment:
New navigation for page table of contents 
<img width="1726" alt="Screenshot 2023-09-10 at 10 10 22 PM" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/58053149/adec0ffb-95f1-41da-90af-1900aee23587">

Reply:
This is part of the updated storybook version. I removed the update changes since it created a big PR and created a separate story for storybook update